lineax/_operator.py

```diff
@@ -1557,6 +1663,36 @@ def _(operator):
     return True
 
 
+@ft.singledispatch
+def is_Woodbury(operator: AbstractLinearOperator) -> bool:
+    """Returns whether an operator is marked as Woodbury.
+    See [the documentation on linear operator tags](../api/tags.md) for more
+    information.
+    **Arguments:**
+    - `operator`: a linear operator.
+    **Returns:**
+    Either `True` or `False.`
+    """
+    _default_not_implemented("is_Woodbury", operator)
```

Comment:
Likewise I don't think this function is needed. Just an `isinstance(x, WoodburyLinearOperator` suffices!

(This kind of compile-time dispatch based on type is a super common design pattern in Equinox-and-friends. See [here](https://docs.kidger.site/equinox/pattern/) for a few other notes on this kind of thing.)